src/default_authorized_user.rs

```diff
@@ -89,7 +89,12 @@ impl ServiceAccount for ConfigDefaultCredentials {
         Some(self.token.read().unwrap().clone())
     }
 
-    async fn refresh_token(&self, client: &HyperClient, _scopes: &[&str]) -> Result<Token, Error> {
+    async fn refresh_token(
+        &self,
+        client: &HyperClient,
+        _scopes: &[&str],
```

Comment:
```suggestion
        _: &[&str],
```
Just a habit really, it doesn't matter much in this case, but... normally it will be dropped quicker when name is just `_`

Reply:
I like it to, but did not want to change style that is already in place.